src/commands/email/send.js

```diff
@@ -37,18 +43,21 @@ class Send extends BaseCommand {
       html: '<p>' + this.emailText + '</p>'
     };
 
-    const fileInfo = await readFileOrStdIn(this.flags.attachment);
+    if (!this.flags[FLAGS.noAttachment]) {
```

Comment:
Why add another flag to indicate no attachment instead of checking whether `this.flags.attachment` is empty?

Reply:
Maybe add a default argument to the flag of `""`?

Reply:
I'd rather be explicit. If you're specifying a `--attachment`, it must be a path to a file. If you don't want an attachment, use `--no-attachment`. If neither is provided, you'll be prompted (existing behavior).